public/js/notifications.js

```diff
@@ -8,25 +8,25 @@ const VERSION = 1;
 
     const LOG_PREFIX = '[Notification] - ';
 
-    if (! 'ServiceWorker' in self) {
+    if (! ('ServiceWorker' in self)) {
         console.error(LOG_PREFIX + "this browser does not support the 'Service Worker API' in the current context");
 
         return;
     }
 
-    if (! 'Navigator' in self) {
+    if (! ('Navigator' in self)) {
         console.error(LOG_PREFIX + "this browser does not support the 'Navigator API' in the current context");
 
         return;
     }
 
-    if (! navigator.serviceWorker) {
+    if (! ('serviceWorker' in navigator)) {
         console.error(LOG_PREFIX + "this browser does not support the 'Service Worker API' in the current context");
 
         return;
     }
```

Comment:
This seems obsolete now, since `! ('ServiceWorker' in self)` is fixed and matches.